src/test/resources/data/custom_advancement_rewards_test/advancements/test/root.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
As AdvancementProviders now work, you may want to make this a generated test to make sure the serialization works as expected.